eftcamb/04p3_power_law_parametrizations_1D.f90

```diff
@@ -0,0 +1,225 @@
+!----------------------------------------------------------------------------------------
+!
+! This file is part of EFTCAMB.
+!
+! Copyright (C) 2013-2016 by the EFTCAMB authors
+!
+! The EFTCAMB code is free software;
+! You can use it, redistribute it, and/or modify it under the terms
+! of the GNU General Public License as published by the Free Software Foundation;
+! either version 3 of the License, or (at your option) any later version.
+! The full text of the license can be found in the file eftcamb/LICENSE at
+! the top level of the EFTCAMB distribution.
+!
+!----------------------------------------------------------------------------------------
+
+!> @file 04p3_power_law_parametrizations_1D.f90
+!! This file contains the definition of the power law parametrization,
+!! inheriting from parametrized_function_1D.
+
+
+!----------------------------------------------------------------------------------------
+!> This module contains the definition of the power law parametrization,
+!! inheriting from parametrized_function_1D.
+
+!> @author Bin Hu, Marco Raveri, Simone Peirone
+
+module EFTCAMB_power_law_parametrizations_1D
+
+    use precision
+    use EFTDef
+    use AMLutils
+    use EFTCAMB_cache
+    use EFTCAMB_abstract_parametrizations_1D
+
+    implicit none
+
+    private
+
+    public power_law_parametrization_1D
+
+    ! ---------------------------------------------------------------------------------------------
+    !> Type containing the power law function parametrization. Inherits from parametrized_function_1D.
+    type, extends ( parametrized_function_1D ) :: power_law_parametrization_1D
+
+        real(dl) :: power_law_value_1
+        real(dl) :: power_law_value_2
```

Comment:
I would use a more informative name that immediately tells people that this is the exponent. What do you think?